### ISSUE TITLE
Use the ExpandingDot wihtout the FLatList 

### ISSUE THREAD

**Issue by darkholiday** [weahforsage/react-native-animated-pagination-dots#8]
Hello 
I would like to pass MaxPages and currentPage to the ExpandingDot component and control the pagination programmatically by updating the currentPage value.

I have tried to add this custom feature but without success.
Can you please give me some hints that can help me proceed.

I will post the final code if i achieved it here.

Thanks.

**Comment by weahforsage** [comment]
> Hello 
> I would like to pass MaxPages and currentPage to the ExpandingDot component and control the pagination programmatically by updating the currentPage value.
> 
> I have tried to add this custom feature but without success.
> Can you please give me some hints that can help me proceed.
> 
> I will post the final code if i achieved it here.
> 
> Thanks.

Yes, but need to rework for this. But if you increase scrollX value by witdh * activeIndex, it should work.

**Comment by darkholiday** [comment]
I tried to change the value of scrollX value with different methods but its didnt apply any change on the ExpandingDot Component.
